glusterd2/transactionv2/txnmanager.go

```diff
@@ -229,16 +224,11 @@ func (tm *txnManager) GetTxnByUUID(id uuid.UUID) (*Txn, error) {
 
 // GetTxns returns all txns added to the store
 func (tm *txnManager) GetTxns() (txns []*Txn) {
-	resp, err := store.Get(context.TODO(), tm.getStoreKey(), clientv3.WithPrefix())
+	resp, err := store.Get(context.TODO(), tm.getStoreKey(txnPrefix), clientv3.WithPrefix())
```

Comment:
You should be using `TxnPrefix` with `getStoreKey` here. This works right now because of a `txnPrefix` defined in transaction.go. You should remove that and make use of a common constant.

Reply:
done